### ISSUE TITLE
when doctor dies and you call a new one your charged for all other doctors that died as well 

### ISSUE THREAD

**Issue by James-the-Coding-Gamer** [BurntJannings/legacy_aimedic#3]
my [Bug] is when a doctor dies and you call a new one your charged for all other doctors that died as well ..

 
and is there a way to set it so it will still res someone even with having enough money  and just put them in the negative balance they have to re pay to get in the positive balance again ?  might be a good enhancement for the script

**Comment by James-the-Coding-Gamer** [comment]
hope this is something can be fixed soon would love to use this with my friends but they dont want to be charged for every dead doc just the one that res you love your scripts sofar though !

**Comment by James-the-Coding-Gamer** [comment]
hope to see a update soon ... @BurntJannings

**Comment by BurntJannings** [comment]
Hey! Good news actually! Update is planned and coming hopefully tomorrow, I have it only charging for one use of doctor correctly, and I've added in my other medic script to make it all one, you'll be able to check pulse and last wound

**Comment by BurntJannings** [comment]
@RealStonerGamer

**Comment by James-the-Coding-Gamer** [comment]
oh so you are making it a full med job script? cuz we use one already we like cuz we can spawn horses and wagons we set in the config in the menu the docs can use and also rez or just bandage as long as we have bandages in our pockets but we do not have the last wound or the check pulse that's Kool

**Comment by BurntJannings** [comment]
Yeah so had all this planned other then horse and wagon, if this is something you'd like, could even add option in this too if you'd like it

**Comment by James-the-Coding-Gamer** [comment]
would love it if you could im sure lots others would to

**Comment by BurntJannings** [comment]
this has been fixed in the new and improved version with more features to come, check here https://github.com/BurntJannings/legacy_medic

**Comment by BurntJannings** [comment]
@RealStonerGamer

**Comment by James-the-Coding-Gamer** [comment]
alright ill test it now
